src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -134,3 +134,20 @@ Errors: Error 0-14: Invalid argument type. Expecting one of the following: Numbe
 >> Value(Index(ParseJSON("{""a"": 5, ""b"": [{""c"": 19 }, {""c"": ""foo"" }] }").b, 1).c)
 19
 
+>> DateDiff(DateValue(ParseJSON("""2011-01-15""")) , DateValue(ParseJSON("""2011-01-30""")))
```

Comment:
error cases of valid dates but not in iso format?

Reply:
Done